public/locales/en/common.json

```diff
@@ -36,7 +34,7 @@
   },
   "timeout-modal": {
     "header": "Your Session Will End Soon",
-    "warning": "To protect your information, you will be logged out in {{count}} minute if you do not continue",
+    "warning": "To protect your information, you will be logged out in {{count}} minute if you do not continue.",
```

Comment:
Should this be minutes, or are there times when it would be 1 minute?

Reply:
Let's just go with plural minutes here since it's only singular for... one minute

Reply:
we _should_ be able to use [i18n pluralization](https://www.i18next.com/translation-function/plurals) but it was giving me trouble. It might be worth another try though since Rocket solved a bug in my i18n setup!

Reply:
wait, there is the plural down below on the next line! it properly pluralizes:

![image](https://user-images.githubusercontent.com/6129479/127672523-1ccadc26-cca7-4e19-9fc0-b68fdea20931.png)

![image](https://user-images.githubusercontent.com/6129479/127672598-56d0c011-1936-45b9-b528-074204b362d2.png)

Reply:
Oh cool, didn't realize we pluralized, thanks for checking Kat!